cmd/cloudFoundryDeploy.go

```diff
@@ -783,7 +783,7 @@ func handleMtaExtensionCredentials(extFile string, credentials map[string]interf
 }
 
 func toEnvVarKey(key string) string {
-	key = regexp.MustCompile(`[^A-Za-z0-9]`).ReplaceAllString(key, "_")
+	key = regexp.MustCompile(`[^A-Za-z0-9]`).ReplaceAllLiteralString(key, "_")
```

Comment:
here it does not really matter, but since we do not expect expansion we should IMO use the `literal` version.